pages/contacts.html

```diff
@@ -0,0 +1,59 @@
+<div class="container-fluid">
+      <div class="row" id="contacts">
+          
+            <div class="col-xs-12 col-sm-12 col-md-12 text-center" style="margin-top: 5%">
+                <h1 class="head-one">CONTACTS</h1>                          
+            </div>
+            
+         <div class="col-md-4 col-md-offset-1 col-sm-9 col-sm-offset-1 col-xs-12" style="float: left"> 
+             <div class="col-xs-12 col-sm-12 col-md-12 text-center" >
+                <p style="">
+                Please feel free to reach out to me if you have any questions, comments of offers.
+                </p>
+                <h2 class="head-two" style="margin-top: 20px; margin-bottom: 20px">CONTACT INFORMATION</h2>
+             </div>
+                
+            
+             <div class="col-md-12 col-sm-12 col-xs-12" style="margin-bottom: 40px">
+                <ul>
+                    <li><img class="icons" src="/asset/img/location.png" alt="location"> Boise ID USA</li>
+                    <li><img class="icons" src="/asset/img/email.png" alt="location"> radkovsn@gmail.com</li>
+                    <li><img class="icons" src="/asset/img/phone.png" alt="location"> (208) 283-9080</li>
+                </ul>
+             </div>
+             
+             <div class="col-md-12 col-sm-12 col-xs-12 text-center" style="margin-bottom: 50px;">
+                <div class="soc-icons" ng-repeat="soclink in $ctrl.socialLinks">
+                     <a href={{soclink.url}} target="_blank">
+                     <img title= {{soclink.title}} ng-src={{soclink.img}} alt={{soclink.title}}>
+                     </a>
+                </div>
+             </div>
+       </div>    
+            <!-- CONTACT FORM -->
+       <div class="col-md-4 col-md-offset-1 col-xs-10 col-xs-offset-1" style="margin-bottom: 100px"> 
+                <form class="form-horizontal" role="form" method="post" action="app.js">
+                    
+                    <div class="form-group">
+                       <label for="name">Full Name</label>
+                       <input type="text" class="form-control" id="name" placeholder="Your Full Name">
+                    </div>
+                      
+                    <div class="form-group">
+                       <label for="email">Email address</label>
+                       <input type="email" class="form-control" id="email" placeholder="Your Email">
+                    </div>
+       
+                    <div class="form-group">
+                        <label for="message">Your Message</label>
```

Comment:
The "for" attribute should point to element id, not the "name" attribute, please, add the id to `<textarea>` matching the "for" attribute.